app/controllers/api/v2/moves_actions.rb

```diff
@@ -3,10 +3,14 @@ module MovesActions
     def index_and_render
       moves = Moves::Finder.new(filter_params, current_ability, params[:sort] || {}).call
 
-      paginate moves,
-               each_serializer: ::V2::MoveSerializer,
-               include: included_relationships,
-               fields: ::V2::MoveSerializer::INCLUDED_FIELDS
+      if csv?
```

Comment:
I was thinking if we could use the standard rails way:
```
respond_to do |format|
...
end
```
and using format.csv and format.json
The advantage would be removing the conditional and a more standard rails way.

Reply:
We don't use plain json format? Curious about how that works for json api mime type

Reply:
I could be wrong but I thought that `respond_to` / `respond_with` relies on routes defining a format (i.e. a file extension) - so that wouldn't work with our approach of using `Accept` header to determine the required type.